libseccomp/src/lib.rs

```diff
@@ -29,18 +29,20 @@
 //!     Ok(())
 //! }
 //! ```
-
```

Comment:
Could you replace these two examples with the following?

```rust
 //! The libseccomp crate is a high-level safe API for the libseccomp library. 
 //!                                                                           
 //! # Examples                                                                
 //!                                                                           
 //! ```rust                                                                   
 //! use libseccomp::*;                                                        
 //!                                                                           
 //! fn main() -> Result<(), Box<dyn std::error::Error>> {                     
 //!     let mut filter = ScmpFilterContext::new(ScmpAction::Allow)?;          
 //!     let syscall = ScmpSyscall::from_name("getuid")?;                      
 //!                                                                           
 //!     filter.add_arch(ScmpArch::X8664)?;                                    
 //!     filter.add_rule(ScmpAction::Errno(1), syscall)?;                      
 //!     filter.set_ctl_log(true)?;                                            
 //!     filter.set_syscall_priority(syscall, 100)?;                           
 //!     filter.load()?;                                                       
 //!                                                                           
 //!     Ok(())                                                                
 //! }                                                                         
 //! ```                                                                       
 //!                                                                           
 //! The above example can be replaced with builder pattern.                   
 //!                                                                           
 //! ```rust                                                                   
 //! use libseccomp::*;                                                        
 //!                                                                           
 //! fn main() -> Result<(), Box<dyn std::error::Error>> {                     
 //!     let syscall = ScmpSyscall::from_name("getuid")?;                      
 //!                                                                           
 //!     ScmpFilterContext::new(ScmpAction::Allow)?                            
 //!         .add_arch(ScmpArch::X8664)?                                       
 //!         .add_rule(ScmpAction::Errno(1), syscall)?                         
 //!         .set_ctl_log(true)?                                               
 //!         .set_syscall_priority(syscall, 100)?                              
 //!         .load()?;                                                         
 //!                                                                           
 //!    Ok(())                                                                 
 //! }                                                                         
 //! ```
```